plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -133,12 +135,13 @@ export const Stepper = (stepperProps: StepperProps) => {
   }: {
     formData?: Record<string, JsonValue>;
   }) => {
-    // TODO(blam): What do we do about loading states, does each field extension get a chance
-    // to display it's own loading? Or should we grey out the entire form.
     setErrors(undefined);
+    setIsValidating(true);
 
     const returnedValidation = await validation(formData);
```

Comment:
What happens if this throws? is it guaranteed not too? prolly add a comment in that case